src/Resource/GetSpecificRate.php

```diff
@@ -39,15 +41,6 @@ public function fetch(ImportConnector $connector): \Iterator
             true
         );
 
-        $rate = function () use ($response) {
-            yield [
-                'asset_id_base' => $response['asset_id_base'],
-                'asset_id_quote' => $response['asset_id_quote'],
-                'rate' => $response['rate'],
-                'time' => $response['time'],
-            ];
-        };
-
-        return new SpecificRateRecord($rate(), count($response), $this);
+        return yield $response;
```

Comment:
`return` does nothing here, since `yield $response` has no return value. Should just be `yield $response`.